docs/configuration.md

```diff
@@ -11,20 +11,20 @@ configuration properties as [JSON](config.yml).
 
 ## Basic configuration
 
-| Parameter       | Type              | Mandatory             | Default value      | Description                                       |
-| --------------- | ----------------- | --------------------- | -----------------  | ------------------------------------------------- |
-| port            | [IP]:port         | no                    | 53                 | Port and optional bind ip address to serve DNS endpoint (TCP and UDP). If you wish to specify a specific IP, you can do so such as 192.168.0.1:53. Example: 53, :53, 127.0.0.1:53  |
-| tlsPort         | [IP]:port         | no                    |                    | Port and optional bind ip address to serve DoT DNS endpoint (DNS-over-TLS). If you wish to specify a specific IP, you can do so such as 192.168.0.1:853. Example: 83, :853, 127.0.0.1:853  |
-| httpPort        | int (1 - 65535)   | no                    |                    | HTTP listener port and optional bind ip address . If > 0, will be used for prometheus metrics, pprof, REST API, DoH ...If you wish to specify a specific IP, you can do so such as 192.168.0.1:4000. Example: 4000, :4000, 127.0.0.1:4000 |
-| httpsPort       | int (1 - 65535)   | no                    |                    | HTTPS listener port and optional bind ip address . If > 0, will be used for prometheus metrics, pprof, REST API, DoH... If you wish to specify a specific IP, you can do so such as 192.168.0.1:443 |
```

Comment:
I removed the part about `> 0` because as far as I can tell, 0 isn't treated specially. Maybe that's a bug, in which case I could add the docs back and fix the code.